packages/web3-eth-ens/src/ressources/ABI/Resolver.js

```diff
@@ -34,33 +169,39 @@ var RESOLVER = [
         "name": "ABI",
         "outputs": [
             {
-                "name": "contentType",
+                "name": "",
```

Comment:
Are those return values also unnamed in legacy Dapps?:)

Reply:
Fixed in efd5cbf